docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DockerJDBCIntegrationSuite.scala

```diff
@@ -112,7 +112,7 @@ abstract class DockerJDBCIntegrationSuite
       // Start the container and wait until the database can accept JDBC connections:
       docker.startContainer(containerId)
       jdbcUrl = db.getJdbcUrl(dockerIp, externalPort)
-      eventually(timeout(60.seconds), interval(1.seconds)) {
+      eventually(timeout(200.seconds), interval(200.seconds)) {
```

Comment:
@yhuai    interval to be set as 200 sec, was a requirement to execute it fine in our local environment of oracle on docker.Less than that it causes exception related to early hit. Since the current test case is ignored, we may revert it back to 60,1; adding a comment here saying to increase this number, when testing with Oracle. Please let me know your thoughts.